client/src/components/ProfileDetailPanel.js

```diff
@@ -21,22 +22,25 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-const ProfileDetailPanel = ({ imageUrl, name, location, about, expertise, buttonType }) => {
-  
-    const classes = useStyles();
-  
-    return (
-      <Paper className={classes.paper}>
-        <Avatar className={classes.avatar} src={imageUrl || null} />
-        <Typography variant="h3">{name}</Typography>
-        <Typography variant="body1">{location}</Typography>
-        <Button type="submit" variant="outlined" color="primary">
-          {buttonType}
-        </Button>
-        <Typography variant="body1">{about}</Typography>
-        <Typography variant="body1">{expertise}</Typography>
-      </Paper>
-    );
+const ProfileDetailPanel = ({ id, imageUrl, name, location, about, expertise, buttonType }) => {
+
+  const classes = useStyles();
+  return (
+    <Paper className={classes.paper}>
+      <Avatar className={classes.avatar} src={imageUrl || null} />
+      <Typography variant="h3">{name}</Typography>
+      <Typography variant="body1">{location}</Typography>
+      <Button type="submit" variant="outlined" color="primary">
+        {
+          buttonType === 'edit' ?
+            <Link to={`/profile/edit/${id}`}>Edit</Link>
+            : buttonType
+        }
```

Comment:
I kind of like to have the conditional on the same like as the { like the following, but personal preference :)
```
{buttonType === 'edit
   ? ...
   : ....
}
```